pkg/steps/artifacts.go

```diff
@@ -25,6 +25,8 @@ import (
 	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/remotecommand"
+	v1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
```

Comment:
please avoid import names like `v1`, they make the code confusing to read :upside_down_face:

Reply:
I blame GoLand ;)